builder/src/build/build-worker.ts

```diff
@@ -100,12 +100,18 @@ globalThis.fetch = (input, init) => {
   if (init) delete init.cache;
   return curFetch(input, init);
 };
+import { Readable } from 'node:stream';
 globalThis.fetch.__nextPatched = isPatchedAlready;
 fetch = globalThis.fetch;
 const CustomRequest = class extends globalThis.Request {
   constructor(input, init) {
     console.log("CustomRequest", input);
-    if (init) delete init.cache;
+    if (init) {
+      delete init.cache;
+      if (init.body?.__node_stream__ === true) {
```

Comment:
I think it should be fine to do without the custom property and just check `instance of Readable`:

```
    if (init) {
      init = {
        ...init,
        cache: undefined,
        body: init.body instanceof Readable ? Readable.toWeb(init.body) : init.body
      }
    }
```

(at least, these works in my tests so far)